docs/index.rst

```diff
@@ -89,5 +89,15 @@ JAX is Autograd_ and XLA_, brought together for high-performance numerical compu
    glossary
 
 
+The core JAX library focuses on scalable array operations.
```

Comment:
WDYT about putting this paragraph _right under_ the "JAX is Autograd and XLA" line, to maximize discoverability? (Also we should probably update that line, since few JAX users remember Autograd at this point...)

Also: I'm not totally sure we need the first sentence. Can we drop it?

Reply:
Done